src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/interceptors/SessionCookieProperties.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.interceptors
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+import java.time.Duration
+
+@ConstructorBinding
+@ConfigurationProperties(prefix = "server.servlet.session.cookie")
+data class SessionCookieProperties(val maxAge: Duration, val httpOnly: Boolean)
```

Comment:
With these annotations (and class) we can easily pull in the matching Spring properties and reference them via an object in the code.